neuro_san/registries/math_guy.hocon

```diff
@@ -54,6 +54,24 @@ on the secret numbers.
                         }
                     },
                     "required": ["operator"]
+                },
+
+                # You can also optionally specify the schema needed for any sly_data.
+                # This is to allow generic clients to prompt for private-data inputs
+                # that do not belong in the chat stream.
+                "sly_data": {
```

Comment:
`expected_sly_data` or `expected_sly_data_schema` would make things less confusing. Otherwise I'd consider this is actual sly data values that are passed to the agent as "constants".

Reply:
Changed to sly_data_schema